decidim-proposals/app/models/decidim/proposals/proposal.rb

```diff
@@ -33,7 +33,8 @@ def self.order_randomly(seed)
       end
 
       def author_name
-        user_group&.name || author&.name || I18n.t("decidim.proposals.models.proposal.fields.official_proposal")
+        return I18n.t("decidim.proposals.models.proposal.fields.official_proposal") if official?
+        user_group&.name || author&.name
```

Comment:
I think you no longer need the `&.` operator in `author&.name` and can use `author.name`.